src/components/model/ModelForm.svelte

```diff
@@ -91,6 +106,29 @@
   function onClearOwner() {
     owner = null;
   }
+
+  function onRetriggerModelExtraction() {
+    if (modelId != null) {
+      const prevLogs = modelLogs;
+
+      modelLogs = {
+        refresh_activity_type_logs: [{ error: null, error_message: null, pending: true, success: false }],
+        refresh_model_parameter_logs: [{ error: null, error_message: null, pending: true, success: false }],
+        refresh_resource_type_logs: [{ error: null, error_message: null, pending: true, success: false }],
+      } as Pick<
+        ModelSlim,
+        'refresh_activity_type_logs' | 'refresh_model_parameter_logs' | 'refresh_resource_type_logs'
+      >;
+
+      // introduce delay for UX feedback
```

Comment:
Delay still needed?

Reply:
Yeah, it's just a little delay to allow the user to visually see a change because sometimes the extraction is instantaneous.

Reply:
Got it, could maybe tweak the comment, interpreted it as needing further feedback